diff --git a/auto_entitylabel.module b/auto_entitylabel.module
index e07a1e4..877f3cf 100644
--- a/auto_entitylabel.module
+++ b/auto_entitylabel.module
@@ -280,7 +280,8 @@ function auto_entitylabel_set_title(&$entity, $type) {
     $titles[LANGUAGE_NONE] = t('@bundle', array('@bundle' => $bundle));
   }
 
-  drupal_alter('auto_entitylabel_title', $titles, clone $entity);
+  $clone = clone $entity;
+  drupal_alter('auto_entitylabel_title', $titles, $clone);
 
   // Ensure the generated title isn't too long.
   $max_length = auto_entitylabel_max_length($type, $settings['title']);
